tabs/src/translations/zh-CN.json

```diff
@@ -1,15 +1,70 @@
 {
     "translation": {
+
+        "Query Runner": {
+            "Key": "(zh-cn) Key",
+            "Value": "(zh-cn) Value",
+            "Run query": "(zh-cn) RUN QUERY",
+            "Request body": "(zh-cn) Request Body",
+            "Request headers": "(zh-cn) Request Headers",
+            "Response body": "(zh-cn) Response Body",
+            "Response headers": "(zh-cn) Response Headers",
+            "Response": "(zh-cn) Response",
+            "Warning": "(zh-cn) To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
```

Comment:
Same suggestion here: "You must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article}."